utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -196,6 +208,18 @@ if [[ ! -d "$DAOS_INSTALL_DIR" ]] ; then
 	fi
 fi
 
+if [[ ! -f "$DAOS_INSTALL_DIR/bin/activate" ]] ; then
+	info "Installing avocado"
+	run $PYTHON_EXE -m venv "$DAOS_INSTALL_DIR"
+	source "$DAOS_INSTALL_DIR/bin/activate"
+	run $PIP_EXE install --upgrade pip
+	run $PIP_EXE install -r "$DAOS_SOURCE_DIR/requirements.txt"
+	deactivate
+	info "Customize venv for DAOS install"
+	run rm -fv "$DAOS_INSTALL_DIR/lib64"
+	run mkdir -pv "$DAOS_INSTALL_DIR/lib64"
```

Comment:
What is this bit doing?

Reply:
It would be good to keep these separate if we can.  They may be some issues with `pydaos.raw` doing that however, but I think you address that with `PYTHONPATH` above?

Reply:
- [ ] Try to split virtualenv and DAOS local install

Reply:
> * [x]  Split virtualenv and DAOS local install

Fixed with commit 79615a794810dfe810a571416598c8cf8f9584b4